test/integration/apiserver/apiserver_test.go

```diff
@@ -3268,6 +3273,279 @@ func TestDisableEmulationVersion(t *testing.T) {
 	}
 }
 
+func TestFeatureGateCompatibilityEmulationVersion(t *testing.T) {
+	tcs := []struct {
+		emulationVersion                          string
```

Comment:
I'm not sure I understand the suggestion here.  Currently this `emulationVersion` value is the value to pass as the value for `--emulated-version=<emulated-version>` and it is a psuedo test-name as this is testing feature gate compatiblity for that version.  I'm happy to make any name change, etc

Reply:
to obtain the source of truth we ran the cluster at binary version == emulation version? I was envisioning what this will look like in the future where we could have a test grid consisting of multiple binary versions emulating the same version, but this is probably fine.

Reply:
Talked offline - the current naming convention seems ok for now.  Marking resolved